### PR TITLE
improve(affiliates): dev mining rewards for 21-01-25 to 21-02-01

### LINKED ISSUE #2490: improve(affiliates): add dev mining payout for week of 2/1
**Why**

EMP deployer rewards from 2020-01-25 23:00:00 to 2021-02-01 23:00:00 utc. AMPLUSD has been added.

Contract | EMP Address | Payout Address
-- | -- | --
pxUSD_MAR2021 | 0x3a93E863cb3adc5910E6cea4d51f132E8666654F | 0x53911776641d6dF38B88b9eF27f920c617E3Cb5e
uGAS-JAN21 | 0x516f595978D87B67401DaB7AfD8555c3d28a3Af4 | 0x97990B693835da58A281636296D2Bf02787DEa17
YD-BTC-MAR21 | 0x1c3f1A342c8D9591D9759220d114C685FD1cF6b8 | 0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35
YD-ETH-MAR21 | 0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5 | 0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35
Zelda "Spring Nuts" USDC | 0xeAddB6AD65dcA45aC3bB32f88324897270DA0387  | 0xF53FBC3aDbe91E1A408716C0AF4C0cDC17A2cca8
Zelda "Summer Nuts" USDC | 0xf215778f3a5e7ab6a832e71d87267dd9a9ab0037 | 0xb8E237CC8D2E81b7C9C9C4B5877200f85100fC87
Zelda "Elastic" USDC | 0x267D46e71764ABaa5a0dD45260f95D9c8d5b8195 | 0x531B290230b1809d0AE29Ea834EdF96748F49Bf9
Zelda Whirlwind Cash |0x2862a798b3defc1c24b9c0d241beaf044c45e585 | 0xa1eD088d0789e7baBa18Ed4d87baca65376850d8
Zelda Reinforced Cash | 0xd81028a6fbaaaf604316f330b20d24bfbfd14478 | 0xdB0d599C3D49F6B349c83Dc2407a16F2830d6fCc
Mini Mario Cash  | 0x7c4090170aeADD54B1a0DbAC2C8D08719220A435 | 0x7eb1e51351380E67F641B84545bf41eCdc10298E
Mario Mini Spring Cache | 0xaD3cceebeFfCdC3576dE56811d0A6D164BF9A5A1 | 0x7eb1e51351380E67F641B84545bf41eCdc10298E
ALTDOM-MAR2021 | 0xC843538d70ee5d28C5A80A75bb94C28925bB1cf2 | 0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E
BTCDOM-MAR2021 | 0xeFA41F506EAA5c24666d4eE40888bA18FA60a1c7 | 0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E
AMPLUSD | 0xab3aa2768ba6c5876b2552a6f9b70e54aa256175 | 0x45Ea614a7Ea47Ec393BDA310F901702DB1347df9

Several of these will force default prices to 1
- 0xeAddB6AD65dcA45aC3bB32f88324897270DA0387
- 0xf215778f3a5e7ab6a832e71d87267dd9a9ab0037
- 0x267D46e71764ABaa5a0dD45260f95D9c8d5b8195
- 0x2862a798b3defc1c24b9c0d241beaf044c45e585
- 0xd81028a6fbaaaf604316f330b20d24bfbfd14478

### PR DESCRIPTION
**Pull request by daywiss**
Signed-off-by: David Adams <david@umaproject.org>

**Motivation**
#2490

**Summary**

Runs dev mining for week of 2/1

**Details**
This week runs rewards, but also adds small patch to handle dispute events, as there was a dispute last week. We see several projects growing in size, particularly the mario/zelda contracts. 

In order to run create a config.json with this data:
```
{
    "empWhitelist": [
      ["0x3a93E863cb3adc5910E6cea4d51f132E8666654F", "0x53911776641d6dF38B88b9eF27f920c617E3Cb5e"],
      ["0x516f595978D87B67401DaB7AfD8555c3d28a3Af4", "0x97990B693835da58A281636296D2Bf02787DEa17"],
      ["0x1c3f1A342c8D9591D9759220d114C685FD1cF6b8", "0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35"],
      ["0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5", "0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35"],
      ["0xeAddB6AD65dcA45aC3bB32f88324897270DA0387", "0xF53FBC3aDbe91E1A408716C0AF4C0cDC17A2cca8"],
      ["0xf215778f3a5e7ab6a832e71d87267dd9a9ab0037", "0xb8E237CC8D2E81b7C9C9C4B5877200f85100fC87"],
      ["0x267D46e71764ABaa5a0dD45260f95D9c8d5b8195", "0x531B290230b1809d0AE29Ea834EdF96748F49Bf9"],
      ["0x2862a798b3defc1c24b9c0d241beaf044c45e585", "0xa1eD088d0789e7baBa18Ed4d87baca65376850d8"],
      ["0xd81028a6fbaaaf604316f330b20d24bfbfd14478", "0xdB0d599C3D49F6B349c83Dc2407a16F2830d6fCc"],
      ["0x7c4090170aeADD54B1a0DbAC2C8D08719220A435", "0x7eb1e51351380E67F641B84545bf41eCdc10298E"],
      ["0xaD3cceebeFfCdC3576dE56811d0A6D164BF9A5A1", "0x7eb1e51351380E67F641B84545bf41eCdc10298E"],
      ["0xC843538d70ee5d28C5A80A75bb94C28925bB1cf2", "0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E"],
      ["0xeFA41F506EAA5c24666d4eE40888bA18FA60a1c7", "0xbca9B2e6B6620197aBA4fdb59079d3FeE21c361E"],
      ["0xab3aa2768ba6c5876b2552a6f9b70e54aa256175", "0x45Ea614a7Ea47Ec393BDA310F901702DB1347df9"]
    ],
    "startTime": 1611615600000,
    "endTime": 1612220400000,
    "totalRewards": "50000",
    "network": 1,
    "fallbackPrices": [
      ["0xeAddB6AD65dcA45aC3bB32f88324897270DA0387", "1"],
      ["0xf215778f3a5e7ab6a832e71d87267dd9a9ab0037", "1"],
      ["0x267D46e71764ABaa5a0dD45260f95D9c8d5b8195", "1"],
      ["0x2862a798b3defc1c24b9c0d241beaf044c45e585", "1"],
      ["0xd81028a6fbaaaf604316f330b20d24bfbfd14478", "1"]
    ]
  }
```
`requires node 14`
Then run `node apps/DevMiningRewards.js config.json --network=mainnet_mnemonic`
**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [x]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

**Issue(s)**

Fixes #2490

### CODE REVIEW

File: packages/affiliates/libs/processors.js [hunk 1]
```diff
@@ -162,6 +162,9 @@ function EmpBalances(handlers = {}, { collateral, tokens } = {}) {
     LiquidationDisputed(/* caller, originalExpirationTimestamp, shutdownTimestamp*/) {
       // nothing
     },
+    DisputeSettled() {
```
Comment (daywiss) [inline packages/affiliates/libs/processors.js:165]: this was such a tiny change i included it in rewards. it does not change output at all.

**Review by smb2796 — APPROVED** [review]
Looks good.

**Review by chrismaree — APPROVED** [review]
looks awesome! the numbers on YD make sense relative to the other projects. I love seeing new EMPs each week!

**Review by hlambur — APPROVED** [review]
LGTM

**Comment by hlambur** [comment]
Sent with tx: https://etherscan.io/tx/0xd7455d3f3f36129898039bde8046db5b7926ae09b4ad2de84bce9a4552ec23a2

**Pull request merged**